### PR TITLE
add info on platformXXX parameters to project create

### PR DESCRIPTION
**Pull request by maxandersen**

### CODE REVIEW

File: docs/src/main/asciidoc/maven-tooling.adoc [hunk 1]
```diff
@@ -67,6 +79,24 @@ If the directory already exists, the generation fails.
 A pair of Dockerfiles for native and jvm mode are also generated in `src/main/docker`.
 Instructions to build the image and run the container are written in those Dockerfiles.
 
+== Using platform
+
+By default the Maven plugin currently uses the extensions that comes with Quarkus core. It is possible to get access to the full ecosystem
```
Comment (emmanuelbernard) [inline docs/src/main/asciidoc/maven-tooling.adoc]: it's only for the create operation, better clarify

**Review by emmanuelbernard — CHANGES_REQUESTED** [review]

File: docs/src/main/asciidoc/maven-tooling.adoc (hunk 1 above)
Comment (maxandersen) [reply docs/src/main/asciidoc/maven-tooling.adoc]: fixed.

File: docs/src/main/asciidoc/maven-tooling.adoc [hunk 2]
```diff
@@ -56,6 +56,18 @@ The following table lists the attributes you can pass to the `create` command:
 | _[]_
 | The list of extensions to add to the project (comma-separated)
 
+| `platformGroupId`
+| `io.quarkus`
+| The group id of the platform you want to use.
+
+| `platformArtifactId`
+| `quarkus-bom`
```
Comment (gsmet) [inline docs/src/main/asciidoc/maven-tooling.adoc]: It should be `quarkus-universe-bom` now?

File: docs/src/main/asciidoc/maven-tooling.adoc [hunk 3]
```diff
@@ -56,6 +56,18 @@ The following table lists the attributes you can pass to the `create` command:
...
+| The artifact id of the platform you want to use.
+
+| `platformVersion`
+| `quarkus-bom`
```
Comment (gsmet) [inline docs/src/main/asciidoc/maven-tooling.adoc]: This is incorrect. It should be a version (or probably more something saying how the version is resolved).

File: docs/src/main/asciidoc/maven-tooling.adoc [hunk 4]
```diff
@@ -67,6 +79,26 @@ If the directory already exists, the generation fails.
 A pair of Dockerfiles for native and jvm mode are also generated in `src/main/docker`.
 Instructions to build the image and run the container are written in those Dockerfiles.
 
+== Using platform
+
+By default at project creation only the extensions that comes with Quarkus core are enabled. It is possible to get access to the full ecosystem
```
Comment (gsmet) [inline docs/src/main/asciidoc/maven-tooling.adoc]:
```suggestion
By default at project creation only the extensions that come with Quarkus core are enabled. It is possible to get access to the full ecosystem
```

File: docs/src/main/asciidoc/maven-tooling.adoc [hunk 5]
```diff
@@ -67,6 +79,26 @@ If the directory already exists, the generation fails.
...
+available on `code.quarkus.io` by using the platform called `universe`.
```
Comment (gsmet) [inline docs/src/main/asciidoc/maven-tooling.adoc]: This is not true anymore.

File: docs/src/main/asciidoc/maven-tooling.adoc [hunk 6]
```diff
@@ -67,6 +79,26 @@ If the directory already exists, the generation fails.
...
+mvn io.quarkus:quarkus-maven-plugin:{quarkus-version}:create \
+    -DprojectGroupId=my-groupId \
+    -DprojectArtifactId=my-artifactId \
+    -DprojectVersion=my-version \
+    -DplatformArtifactId=quarkus-universe-bom \
+    -DplatformVersion={quarkus-version} \
+    -DclassName="org.my.group.MyResource"
+----
+
+Once a project are created using a platform the operations dealing with extensions will use that automatically.
```
Comment (gsmet) [inline docs/src/main/asciidoc/maven-tooling.adoc]:
```suggestion
Once a project is created using a platform, the operations dealing with extensions will use that automatically.
```

**Review by gsmet — CHANGES_REQUESTED** [review]
This PR is interesting but oudated. Could you please check my comments and suggestions?

Thanks.

**Comment by maxandersen** [comment]
this is already merged into master thus closing.

**Pull request closed without merging**